cmake/scripts/helpers.cmake

```diff
@@ -0,0 +1,90 @@
+get_filename_component(ROOT_DIR "${CMAKE_CURRENT_LIST_DIR}/../.." ABSOLUTE)
+
+set(CACHE_DIR "${ROOT_DIR}/deps/src")
+set(PACKAGES_DIR "${ROOT_DIR}/packages")
+
+function(download URL DST_FILE STATUS)
```

Comment:
To be honest, I don't remember. It was my first contact with CMake scripting (this file is copied from cpp-dependencies, only needed by cpp-ethereum on Windows). It looks I added some kind of checking if the file is already there. ExternalProject does it nicely, but pure `file(DOWNLOAD ...)` maybe not.